rafthttp/entry_reader.go

```diff
@@ -20,11 +20,24 @@ import (
 	"encoding/binary"
 	"io"
 
+	"github.com/coreos/etcd/pkg/metrics"
 	"github.com/coreos/etcd/raft/raftpb"
 )
 
 type entryReader struct {
-	r io.Reader
+	r             io.Reader
+	entriesRecv   metrics.Counter
```

Comment:
well. probably we can also remove total. make it consistent with line37 but without received